.pre-commit-config.yaml

```diff
@@ -0,0 +1,10 @@
+repos:
+-   repo: https://github.com/psf/black
+    rev: 22.8.0
```

Comment:
Please run `pre-commit autoupdate --freeze` to store "frozen" hashes in `rev` instead of tag names - this mitigates the unlikely event that the tag is changed

Reply:
Thanks, done.